LSP-copilot.sublime-settings

```diff
@@ -17,7 +17,10 @@
 		"local_checks": false,
 		"telemetry": false,
 		"proxy": "",
-		"completion_style": "popup"
+		"completion_style": "popup",
+		"copilot_ignore": [
+			"**/*.py"
```

Comment:
`fnmatch` doesn't support globs

Reply:
Fwiw, we can use https://github.com/facelessuser/wcmatch if we want. `wcmatch` is already on py38 deps for PC v4. But still, I think it's better to just follow `.copilotignore`.